neqo-transport/src/connection/tests/resumption.rs

```diff
@@ -129,7 +129,7 @@ fn ticket_rtt(rtt: Duration) -> Duration {
     dec.skip_vvec(); // Skip over the payload.
 
     // Replace the ACK frame with PADDING.
-    plaintext[..ACK_FRAME_1.len()].fill(FRAME_TYPE_PADDING.try_into().unwrap());
+    plaintext[..ACK_FRAME_1.len()].fill(FrameType::Padding.into());
```

Comment:
```suggestion
    plaintext[..ACK_FRAME_1.len()].fill(u64::from(FrameType::Padding));
```

Of course, I think that this is now really `u8::try_from()` instead, because this is `&mut [u8]`.